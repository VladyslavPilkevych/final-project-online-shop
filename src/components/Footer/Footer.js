import React from 'react';
// import styles from './Footer.module.scss';

function Header() {
  return (
    <footer></footer>
  );
}

export default Header;
