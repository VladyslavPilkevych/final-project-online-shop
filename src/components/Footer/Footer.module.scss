@import '../../assets/styles/global.module.scss';

.footer {
    background-color: #020203;
    margin-top: 44.5px;
}

.footerWrapper {
    max-width: 1398px;
    margin: 0 auto;
}

.footerTitle {
    font-family: $fonts;
    font-style: normal;
    font-weight: 500;
    font-size: 38px;
    line-height: 132.5%;
    color: #FFFFFF;
    padding-top: 47px;
}

.footerText {
    font-family: $fonts;
    font-style: normal;
    font-weight: 300;
    font-size: 16px;
    line-height: 132.5%;
    color: #FFFFFF;
    margin-top: 9px;
}

.footerListContainer {
    display: flex;
    margin-top: 45px;
    justify-content: space-between;
}

.footerListName {
    font-family: $fonts;
    font-style: normal;
    font-weight: 700;
    font-size: 14px;
    line-height: 100%;
    color: rgba(255, 255, 255, 0.5);
}

.footerListWrapper {
    margin-top: 30px;
    margin-bottom: 90px;
}

.footerListPart {
    font-family: $fonts;
    font-style: normal;
    font-weight: 400;
    font-size: 13px;
    line-height: 140%;
    color: #FFFFFF;
}

.footerBottomWrapper {
    display: flex;
    justify-content: space-between;
}

.footerBottomText {
    font-family: $fonts;
    font-style: normal;
    font-weight: 500;
    font-size: 12px;
    line-height: 132.5%;
    color: #FFFFFF;
    opacity: 0.6;
}

.footerIconsWrapper {
    margin-bottom: 20px;
    position: relative;
}

.footerIconsWrapper::before {
    position: absolute;
    content: "";
    width: 1398px;
    height: 1px;
    background: #FFFFFF;
    opacity: 0.2;
    top: -50%;
}

.faceBoookGrey {
    margin-right: 10px;
}