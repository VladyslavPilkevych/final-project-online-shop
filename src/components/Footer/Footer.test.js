/* eslint-disable react/react-in-jsx-scope */
import { configure, render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import Footer from './Footer';

describe('Snapshot footer should render', () => {
  test('Snapshot footer is rendered', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe('More information should open, after click on arrow', () => {
  test('More info is open', () => {
    render(<Footer />);
    configure({ testIdAttribute: 'id' });
    userEvent.click(screen.getByTestId(1));
    expect(screen.getByText('Information')).toBeInTheDocument();
    screen.debug();
  });
});