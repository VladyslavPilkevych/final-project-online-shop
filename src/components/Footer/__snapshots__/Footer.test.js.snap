// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot footer should render Snapshot footer is rendered 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footerWrapper"
    >
      <p
        class="footerTitle"
      >
        Sign Up To Our Newsletter.
      </p>
      <p
        class="footerText"
      >
        Be the first to hear about the latest offers.
      </p>
      <div
        class="footerListContainer"
      >
        <div
          class="footerList"
        >
          <div
            class="footerListNameWrapper"
          >
            <h3
              class="footerListName"
            >
              Information
            </h3>
            <div
              class="footerArrowWrapper"
              id="1"
            >
              <img
                alt="arrow-icon"
                class="footerArrowDown"
                src="icon-arrow-down.png"
              />
            </div>
          </div>
          <div
            class="footerListWrapperClose"
          >
            <p
              class="footerListPart"
            >
              About Us
            </p>
            <p
              class="footerListPart"
            >
              About Zip
            </p>
            <p
              class="footerListPart"
            >
              Privacy Policy
            </p>
            <p
              class="footerListPart"
            >
              Search
            </p>
            <p
              class="footerListPart"
            >
              Terms
            </p>
            <p
              class="footerListPart"
            >
              Orders and Returns
            </p>
            <p
              class="footerListPart"
            >
              Contact Us
            </p>
            <p
              class="footerListPart"
            >
              Advanced Search
            </p>
          </div>
        </div>
        <div
          class="footerList"
        >
          <div
            class="footerListNameWrapper"
          >
            <h3
              class="footerListName"
            >
              PC Parts
            </h3>
            <div
              class="footerArrowWrapper"
              id="2"
            >
              <img
                alt="arrow-icon"
                class="footerArrowDown"
                src="icon-arrow-down.png"
              />
            </div>
          </div>
          <div
            class="footerListWrapperClose"
          >
            <p
              class="footerListPart"
            >
              CPUS
            </p>
            <p
              class="footerListPart"
            >
              Add On Cards
            </p>
            <p
              class="footerListPart"
            >
              Hard Drives (Internal)
            </p>
            <p
              class="footerListPart"
            >
              Graphic Cards
            </p>
            <p
              class="footerListPart"
            >
              Keyboards / Mice
            </p>
            <p
              class="footerListPart"
            >
              Cases / Power Supplies / Cooling
            </p>
            <p
              class="footerListPart"
            >
              RAM (Memory)
            </p>
            <p
              class="footerListPart"
            >
              Software
            </p>
            <p
              class="footerListPart"
            >
              Speakers / Headsets
            </p>
            <p
              class="footerListPart"
            >
              Motherboards
            </p>
          </div>
        </div>
        <div
          class="footerList"
        >
          <div
            class="footerListNameWrapper"
          >
            <h3
              class="footerListName"
            >
              Desktop PCs
            </h3>
            <div
              class="footerArrowWrapper"
              id="3"
            >
              <img
                alt="arrow-icon"
                class="footerArrowDown"
                src="icon-arrow-down.png"
              />
            </div>
          </div>
          <div
            class="footerListWrapperClose"
          >
            <p
              class="footerListPart"
            >
              Custom PCs
            </p>
            <p
              class="footerListPart"
            >
              Servers
            </p>
            <p
              class="footerListPart"
            >
              MSI All-In-One PCs
            </p>
            <p
              class="footerListPart"
            >
              HP/Compaq PCs
            </p>
            <p
              class="footerListPart"
            >
              ASUS PCs
            </p>
            <p
              class="footerListPart"
            >
              Tecs PCs
            </p>
          </div>
        </div>
        <div
          class="footerList"
        >
          <div
            class="footerListNameWrapper"
          >
            <h3
              class="footerListName"
            >
              Laptops
            </h3>
            <div
              class="footerArrowWrapper"
              id="4"
            >
              <img
                alt="arrow-icon"
                class="footerArrowDown"
                src="icon-arrow-down.png"
              />
            </div>
          </div>
          <div
            class="footerListWrapperClose"
          >
            <p
              class="footerListPart"
            >
              Everyday Use Notebooks
            </p>
            <p
              class="footerListPart"
            >
              MSI Workstation Series
            </p>
            <p
              class="footerListPart"
            >
              MSI Prestige Series
            </p>
            <p
              class="footerListPart"
            >
              Tablets and Pads
            </p>
            <p
              class="footerListPart"
            >
              Netbooks
            </p>
            <p
              class="footerListPart"
            >
              Infinity Gaming Notebooks
            </p>
          </div>
        </div>
        <div
          class="footerList"
        >
          <div
            class="footerListNameWrapper"
          >
            <h3
              class="footerListName"
            >
              Address
            </h3>
            <div
              class="footerArrowWrapper"
              id="5"
            >
              <img
                alt="arrow-icon"
                class="footerArrowDown"
                src="icon-arrow-down.png"
              />
            </div>
          </div>
          <div
            class="footerListWrapperClose"
          >
            <p
              class="footerListPart"
            >
              Address: 1234 Street Adress City Address, 1234
            </p>
            <p
              class="footerListPart"
            >
              Phones: (00) 1234 5678
            </p>
            <p
              class="footerListPart"
            >
              We are open: Monday-Thursday: 9:00 AM - 5:30 PM
            </p>
            <p
              class="footerListPart"
            >
              Friday: 9:00 AM - 6:00 PM
            </p>
            <p
              class="footerListPart"
            >
              Saturday: 11:00 AM - 5:00 PM
            </p>
            <p
              class="footerListPart"
            >
              E-mail: shop@email.com
            </p>
          </div>
        </div>
      </div>
      <div
        class="footerBottomWrapper"
      >
        <div
          class="footerIconsWrapper"
        >
          <svg
            class="faceBoookGrey"
          >
            facebook-icon-grey.svg
          </svg>
          <svg>
            instagram-icon-grey.svg
          </svg>
        </div>
        <p
          class="footerBottomText"
        >
          Copyright Â© 2020 Shop Pty. Ltd.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
